applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -25,8 +25,9 @@ namespace Kratos
 //***********************DEFAULT CONSTRUCTOR******************************************
 //************************************************************************************
 
-NodalConcentratedElement::NodalConcentratedElement( IndexType NewId, GeometryType::Pointer pGeometry )
+NodalConcentratedElement::NodalConcentratedElement( IndexType NewId, GeometryType::Pointer pGeometry, bool UseRayleighDamping )
```

Comment:
Oh, I got you wrong. I just changed it in the header file (like for the ShellThinElement / ShellThinCorotationalElement). Thanks for pointing that out, I will change it.